index.js

```diff
@@ -15,11 +16,20 @@ var nameMap = {
 };
 
 module.exports = function (release) {
-	var version = /\d+\.\d+/.exec(release || os.release());
+	var verRe = /\d+\.\d+/;
+	var version;
 
-	if (!version) {
+	if (release || !semver.satisfies(process.version, '>=0.12.0 <3.1.0')) {
+		version = verRe.exec(release || os.release());
+	} else if (/^win/i.test(process.platform)) {
+		// Workaround for Windows 10 on node < 3.1.0
+		var execSync = require('child_process').execSync;
```

Comment:
Can you put the workaround are a separate `if` check before the normal one? I think that would make the logic clearer. Right now it's a bit hard to read.